_articles/manage-repos.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: article
+title: Manage Sources in Pop!_OS
+description: >
+  Add, remove and edit repositories in Pop!_OS
+keywords:
+  - Pop!_OS 18.04
+  - Pop!_OS 19.10
+  - repoman
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
+
+---
+
+### Accessing Repoman
+
+The tool that Pop!_OS has for adding, removing and editing repositories is called <u>Repoman</u> which is a part of the Pop!_Shop.
```

Comment:
Pop!_OS' tool for adding removing and editing repositories is called Repoman. Repoman is part of the Pop!_Shop.